style.css

```diff
@@ -82,3 +82,32 @@ a:active {
 .wp-block-button__link:hover {
 	opacity: 0.90;
 } 
+
+/*
+ * Alignment styles.
+ * Necessary until the following issues are resolved in Gutenberg:
+ * https://github.com/WordPress/gutenberg/issues/35607
+ * https://github.com/WordPress/gutenberg/issues/35884
+ */
+
+.wp-site-blocks,
+.is-root-container,
+.edit-post-visual-editor__post-title-wrapper {
+	padding-left: var(--wp--custom--spacing--outer);
+	padding-right: var(--wp--custom--spacing--outer);
+}
+
+.wp-site-blocks .alignfull,
+.is-root-container .wp-block[data-align="full"] {
```

Comment:
This basically means all aligned full elements get the negative margins. This breaksif I define a custom layout in say a group inside a column (split 2) and apply a different padding or no padding at all.

Reply:
do you think making an exception for those cases will be enough or can you think of more? Because limiting this to only the first children inside wp-site-blocks does basically nothing in this particular themes, where the actual content that will be alignfull is deeply nested

Reply:
Yeah, this is the reason why this broke up here: https://github.com/WordPress/twentytwentytwo/pull/291#issuecomment-991058271. Those negative margins shouldn't come into play when nested inside of columns.